api/app/tests/apps/signals/test_api_v1_endpoints.py

```diff
@@ -598,6 +598,112 @@ def test_put_not_allowed(self):
         response = self.client.put(detail_endpoint, {}, format='json')
         self.assertEqual(response.status_code, 405)
 
+    def test_split(self):
+        self.client.force_authenticate(user=self.superuser)
+
+        self.assertEqual(Signal.objects.count(), 2)
+
+        pk = self.signal_no_image.id
+        split_endpoint = '{}/split'.format(self.detail_endpoint.format(pk=pk))
+
+        response = self.client.post(
+            split_endpoint,
+            [
+                {'text': 'Child #1'},
+                {'text': 'Child #2'}
+            ],
+            format='json'
+        )
```

Comment:
Should we also test what happens when additional data is passed to the endpoint? To test that it is not possible to override other data.